annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -539,14 +541,17 @@ protected void validateSubtitleOrCaption(Annotation webAnnotation)
     protected void validateLinkForContributing(Annotation webAnnotation)
         throws ParamValidationI18NException, RequestBodyValidationException, PropertyValidationException {
 
-      validateBodyExists(webAnnotation.getBody());
-
-      if (webAnnotation.getTarget() == null) 
+      /* the body can be either a string (having only the value) or a json object with the id key
+       * in which case it will have the httpUri value
+       */
+      if (webAnnotation.getBody() == null 
```

Comment:
better use validateBodyExists instead

Reply:
We added in the ticket description to allow only https URLs in the body. This condition needs to be added both to create and update methods.

Reply:
Makes sense, updated.